sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -968,7 +966,7 @@ public void run() {
 
       // Wait for longer than normal when fetching a batch to improve chances a record is available
       // when start() returns.
-      nextBatch(START_NEW_RECORDS_POLL_TIMEOUT);
```

Comment:
actually, can you remove this arg for nextBatch and use `NEW_RECORDS_POLL_TIMEOUT` diretly inside nextBatch().